### PR TITLE
Release/v0.4.0

### PR DESCRIPTION
**Pull request by bburns632**
This is the initial documentation changes for a [v0.4.0](https://github.com/UptakeOpenSource/pkgnet/milestone/2) release.  After #213 is resolved, I will: 

1. build the docs,  
2. append to this PR
3. test, and 
4. after collective PR approval, submit to CRAN. 

For more information on the release process, see the CRAN section of [CONTRIBUTING.md](https://github.com/UptakeOpenSource/pkgnet/blob/master/CONTRIBUTING.md#cran).

For even more detail, check out the extensions to that section proposed in this PR.

### CODE REVIEW

**Comment by bburns632** [comment]
#213 has been resolved.  This PR is now as it will be submitted (including website update).  @jayqi & @jameslamb, are you able to review this today so as to allow CRAN submission tonight?

**Comment by bburns632** [comment]
@jayqi thanks.  The vignette version of the pkgnet report is now up on the gallery.  With the News update, we should be good to submit to CRAN.

**Comment by bburns632** [comment]
Submitted to CRAN just now.  Pending approval.

File: cran-comments.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+Please fix and resubmit.
+
+
+Is there some reference about the method you can add in the Description 
+field in the form Authors (year) <doi:.....>?
+
+### Resubmission on April 2nd, 2019 (planned)
+The problematic links have been fixed:   
+* two were non-canonical CRAN links and   
+* one was a link to an upcoming gallery exhibit that does not yet exist.
```
Comment (jayqi) [inline cran-comments.md]: The link in NEWS.md is a link to a vignette that is getting created as a part of this release.

File: cran-comments.md [hunk 2]
```diff
@@ -1,5 +1,36 @@
...
+field in the form Authors (year) <doi:.....>?
+
+### Resubmission on April 2nd, 2019 (planned)
+The problematic links have been fixed:   
+* two were non-canonical CRAN links and   
+* one was a link to an upcoming gallery exhibit that does not yet exist.   
+
+I had seen the warnings for the non-canonical CRAN links during testing and had interpreted them as optional.  In future testing, these CRAN link formatting warnings will be interpreted as required.  However, the broken link had gone unnoticed, burred under the other two warnings.  Thanks for the check.  
+
+Regarding the references comment, the package is original work aside from the functionality utilized from the packages listed in the DESCRIPTION file.  However, I will confer with the pkgnet authors on writing a paper of our own, describing our motivations and design decisions in detail (with a full bibliography).  Thanks for the complement!
```
Comment (jayqi) [inline cran-comments.md]: Compliment with an e. Also I don't think this was a compliment, that's just a standard thing they started asking for recently.

File: cran-comments.md [hunk 3]
```diff
@@ -1,5 +1,36 @@
...
+
+Is there some reference about the method you can add in the Description 
+field in the form Authors (year) <doi:.....>?
+
+### Resubmission on April 2nd, 2019 (planned)
+The problematic links have been fixed:   
+* two were non-canonical CRAN links and   
+* one was a link to an upcoming gallery exhibit that does not yet exist.   
+
+I had seen the warnings for the non-canonical CRAN links during testing and had interpreted them as optional.  In future testing, these CRAN link formatting warnings will be interpreted as required.  However, the broken link had gone unnoticed, burred under the other two warnings.  Thanks for the check.
```
Comment (jayqi) [inline cran-comments.md]: "buried". Also the broken link is intentional.

Comment (jameslamb) [inline cran-comments.md]: typo: `burred` --> `buried`

File: R/CreatePackageVignette.R [hunk 4]
```diff
@@ -4,12 +4,9 @@
...
 #'    into HTML vignettes upon package building. It is also compatible with
-#'    \code{\link[pkgdown:build_articles]{pkgdown}} sites. See the vignette
-#'    \href{https://cran.r-project.org/web/packages/pkgnet/vignettes/publishing-reports.html}{
-#'    "Publishing Your pkgnet Package Report"} for details about how to use this
-#'    function, as well as
-#'    \href{https://cran.r-project.org/web/packages/pkgnet/vignettes/pkgnet-report.html}{
-#'    our example for pkgnet}.
+#'    \code{\link[pkgdown:build_articles]{pkgdown}} sites. See examples of these vignette formatted reports as well as standard formatted reports 
+#'    at \href{https://uptakeopensource.github.io/pkgnet-gallery/docs/articles/pkgnet-gallery.html}{the pkgnet gallery}.  Find examples of how to use
+#'    a pkgnet report in the \href{https://uptakeopensource.github.io/pkgnet/articles/index.html}{pkgnet vignettes}.
```
Comment (jayqi) [inline R/CreatePackageVignette.R]: I don't think you should change the text here. Keep the original text, change the first link to 

- https://uptakeopensource.github.io/pkgnet/articles/publishing-reports.html

and the second link to

- https://uptakeopensource.github.io/pkgnet-gallery/exhibits/pkgnet-vignette/pkgnet-vignette.html

**Comment by bburns632** [comment]
Current version passing R CMD check for all but the one link that will be valid upon release.

**Comment by bburns632** [comment]
Just submitted again.

**Comment by bburns632** [comment]
And.... we're up!

**Pull request merged**